pset/pset.go

```diff
@@ -289,9 +289,9 @@ func (p *Pset) SanityCheck() error {
 	return nil
 }
 
-func (p *Pset) ValidateAllSignatures() (bool, error) {
+func (p *Pset) ValidateAllSignatures(net *network.Network) (bool, error) {
```

Comment:
Please revert these changes.
Network has nothing to do with signature verification. As I said in the previous comment, it's totally fine to hardcode either Liquid or Regtest because we are not interested in deriving addresses but just scripts/hashes. Network is useful just to prepend the correct address prefix when encoding the script into base58/bech32/blech32 format.

Reply:
Agreed, better to pass nil, to avoid confusion.

Reply:
Done